knimin/tests/test_auth_handlers.py

```diff
@@ -0,0 +1,66 @@
+from unittest import main
+
+from tornado.escape import url_escape
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+
+
+class TestAuthLoginHandler(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/auth/login/')
+        self.assertEqual(response.code, 200)
+        # make sure redirect happened properly
+        port = self.get_http_port()
+        self.assertEqual(response.effective_url, 'http://localhost:%d/' % port)
+
+    def test_post(self):
+        # check if unkown users are recognized
+        response = self.post('/auth/login/',
+                             {'email': 'idontexist',
+                              'password': 'password',
+                              })
+        self.assertIn('>Unknown user<', response.body)
```

Comment:
I can't foresee which other parts might break due to this modification. Unittest are exactly for this purpose, don't they? So I would prefer to first complete all unit tests and only then modify the code - but we should raise a new issue here.